heroic-core/src/main/java/com/spotify/heroic/metric/LocalMetricManager.java

```diff
@@ -199,6 +201,9 @@ public String toString() {
             private final Span parentSpan;
             private final MetricType source;
 
+            private static final double PROBABILITY = 0.005;
```

Comment:
This should definitely be configurable - possibly using the global probability as the default and having a config option to override it.